File: DSA PRACTICE/LECTURE3/Pattern1.cpp
#include<iostream>
using namespace std;


void  pattern(int n)
    {
        int i=1;
        while(i<=n)
        {
            int j=1;
            
            while(j<=n)
            {
                cout<<"* ";
                j++;
            }
            cout<<endl;
            i++;
        }
    }

int main() 
    {
       int n;
       cout<<"Enter the number"<<endl;
       cin>>n;

       pattern(n);
       return 0;
    }




/*OUTPUT:
Enter the number
4
*****
*****
*****
*****
*/

File: DSA PRACTICE/LECTURE3/Pattern1.java
package patterns;
import java.util.*;

class Pattern1
{

  public static void  pattern(int n)
    {
        int i=1;
        while(i<=n)
        {
            int j=1;
            
            while(j<=n)
            {
                System.out.print("*");
                j++;
            }
            System.out.println();
            i++;
        }
    }

    public static void main(String[] args) 
    {
        Scanner obj=new Scanner(System.in);
        System.out.println("Enter the number");
        int n=obj.nextInt();

        Pattern1.pattern(n);
    }
}



/*OUTPUT:
Enter the number
4
*****
*****
*****
*****
*/